cmd/kubernetes/kubernetes.go

```diff
@@ -81,7 +81,7 @@ func (h *delegateHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 func runApiServer(etcdClient tools.EtcdClient, addr net.IP, port int, masterServiceNamespace string) {
 	handler := delegateHandler{}
 
-	helper, err := master.NewEtcdHelper(etcdClient, "")
+	helper, err := master.NewEtcdHelper(etcdClient, "", "")
```

Comment:
The way the master.NewEtcdHelper is written in the latest patch is: if you call master.NewEtcdHelper with an empty string for the prefix, it defaults to "/registry" so if you were to supply "registry" it would end up as "/registry/registry".

But please let me know if I've misunderstood something.